testframework/tests/RunPQSDKTestSuites.ps1

```diff
@@ -50,57 +50,64 @@ param(
     [string[]]$TestSettingsList,
     [switch]$ValidateQueryFolding,
     [switch]$DetailedResults,
-    [switch]$JSONResults
+    [switch]$JSONResults,
+    [string]$RunPQSDKTestSuitesSettingsPath,
+    [switch]$Pipeline
 )
 
 # Pre-Requisite:   
 # Ensure the credentials are setup for your connector following the instructions here: https://learn.microsoft.com/en-us/power-query/power-query-sdk-vs-code#set-credential 
 
 # Retrieving the settings for running the TestSuites from the JSON settings file
-$RunPQSDKTestSuitesSettings = Get-Content -Path RunPQSDKTestSuitesSettings.json | ConvertFrom-Json
+if ($RunPQSDKTestSuitesSettingsPath) {
```

Comment:
add look for the file in the root folder by default in case the parameter is not set